tests/test_dashboard/test_statistics/test_certificates.py

```diff
@@ -44,3 +46,45 @@ def test_get_certificates_count_not_downloadable(base_data, fx_permission_info):
     ).update(status=some_status_not_downloadable)
     result = certificates.get_certificates_count(fx_permission_info)
     assert result == {'org1': 3, 'org2': 10}, f'Wrong certificates result. expected: {result}'
+
+
+@pytest.mark.django_db
+@override_settings(FX_DEFAULT_COURSE_EFFORT=10)
+@pytest.mark.parametrize('tenant_ids, expected_result', [
+    ([1], 14 * 10),
+    ([2], 9 * 10),
+    ([3], 0 * 10),
+])
+def test_get_learning_hours_count_for_default_course_effort(
+    base_data, fx_permission_info, tenant_ids, expected_result
+):  # pylint: disable=unused-argument
+    """Verify get_learning_hours_count function."""
+    fx_permission_info['view_allowed_full_access_orgs'] = get_tenants_orgs(tenant_ids)
+    fx_permission_info['view_allowed_any_access_orgs'] = get_tenants_orgs(tenant_ids)
+    result = certificates.get_learning_hours_count(fx_permission_info)
+    assert result == expected_result, \
+        f'Wrong learning hours count for tenant(s) {tenant_ids}. expected: {expected_result}, got: {result}'
+
+
+@pytest.mark.django_db
+@override_settings(FX_DEFAULT_COURSE_EFFORT=10)
+@pytest.mark.parametrize('effort, expected_result, usecase', [
+    # 2 is the certificate_count in tenant_id 8
+    ('20:30', 20.5 * 2, 'Valid, proper HH:MM format with 2-digit minutes'),
+    ('20:05', 20.1 * 2, 'Valid, minutes as a single digit with leading zero'),
+    ('20:5', 20.1 * 2, 'Valid, minutes as a single digit without leading zero'),
+    ('5:5', 5.1 * 2, 'Valid, both hours and minutes are single digits'),
+    ('30', 30 * 2, 'Valid, only hours provided (no minutes)'),
+    ('5:120', 10 * 2, 'Invalid, minutes exceed 60, use default value'),
+    ('invalid', 10 * 2, 'Invalid, non-numeric value for hours, use default value'),
+    ('20:invalid', 10 * 2, 'Invalid, non-numeric value for minutes, use default value'),
```

Comment:
two more tests, please
```suggestion
    ('20:invalid', 10 * 2, 'Invalid, non-numeric value for minutes, use default value'),
    ('0:25', 10 * 2, 'Invalid, less than 30 minutes'),
    ('0:-55', 10 * 2, 'Invalid, less than 30 minutes'),
```

Reply:
that's a good catch there.